vertx-web-graphql/src/test/java/io/vertx/ext/web/handler/graphql/MultipartRequestTest.java

```diff
@@ -0,0 +1,111 @@
+package io.vertx.ext.web.handler.graphql;
+
+import graphql.GraphQL;
+import graphql.schema.*;
+import graphql.schema.idl.RuntimeWiring;
+import graphql.schema.idl.SchemaGenerator;
+import graphql.schema.idl.SchemaParser;
+import graphql.schema.idl.TypeDefinitionRegistry;
+import io.vertx.core.http.HttpClient;
+import io.vertx.core.http.HttpClientRequest;
+import io.vertx.core.json.JsonObject;
+import io.vertx.ext.web.FileUpload;
+import io.vertx.ext.web.WebTestBase;
+import io.vertx.ext.web.handler.BodyHandler;
+import org.junit.Test;
+
+import java.util.List;
+
+class Result {
+  private final String fileName;
+
+  Result(final String fileName) {
+    this.fileName = fileName;
+  }
+}
+
+public class MultipartRequestTest extends WebTestBase {
```

Comment:
Can you add a test for:

- same file used in different variables
- multiple files
- multiple operations (batch)